front/src/i18n/en-us/index.js

```diff
@@ -3,5 +3,7 @@
 
 export default {
   failed: 'Action failed',
-  success: 'Action was successful'
+  success: 'Action was successful',
+  defaultTextForNothingHereCmp: 'Nothing here...',
```

Comment:
Can you make this simpler, just "nothingHere"? and change the string to "There's nothing here.".

Please also use i18n for text used in other components that is not default, and instead of storing the string in each file, just send the relevant i18n text.